integration-tests/api/__tests__/admin/sales-channels.js

```diff
@@ -632,7 +621,7 @@ describe("sales channels", () => {
         relations: ["sales_channels"],
       })
 
-      expect(attachedProduct.sales_channels.length).toBe(1)
+      expect(attachedProduct.sales_channels.length).toBe(2)
       expect(attachedProduct.sales_channels).toEqual(
```

Comment:
suggest(non blocking): We should test the second one as well which I suppose should be the default one. Just to ensure that nothing changes the fact that the default sales channel is present instead of any sales channel